cddl/attest-key-triple-record.cddl

```diff
@@ -1,4 +1,8 @@
 attest-key-triple-record = [
-  environment-map
-  [ + $crypto-key-type-choice ]
+  environment: environment-map
+  key-list: [ + $crypto-key-type-choice ]
+non-empty<{
+   ? &(mkey: 0) => $measured-element-type-choice,
+   ? &(authorized-by: 2) => [ + $crypto-key-type-choice ] 
+}>
```

Comment:
Not sure how the suggestion was pulled in. This was:

```suggestion
  ? mkey: $measured-element-type-choice
  ? authority-list: [ + $crypto-key-type-choice ]
```

Reply:
> What was the difference between authorized-by and authority-list?

nothing.